lib/jaeger/client/propagation_codec/b3_codec.rb

```diff
@@ -0,0 +1,93 @@
+# frozen_string_literal: true
+
+module Jaeger
+  module Client
+    module PropagationCodec
+      class B3Codec
```

Comment:
I would convert this to a module and only have static functions here. Don't see much point of creating an instance when it's not needed.